### DISCUSSION TITLE
Partition data loss

### DISCUSSION THREAD

**Discussion by YiXinChenChen** [milvus-io/milvus#22720] [Q&A and General discussion]
I created a collection A and created two partitions p0 and p1 under A. When I write data to p0, In first time search, I can get the result and release A, but when I search for the second time, I cannot find the result, why?

create collection A ->create partition p0 -> insert date in p0 ->load A with p0 ->search(success) ->release A ->load A with p0 ->search(get empty)

**Comment by yhmo** [comment]
First of all, check the collection parameters, **don't** set the "collection.ttl.seconds" like this:
```
    schema = CollectionSchema(fields=[field1, field2], description="collection description")
    collection = Collection(name=name, data=None, schema=schema, properties={"collection.ttl.seconds": 15})
```
With this parameter, milvus will automatically delete data after 15 seconds.
